lerobot/common/datasets/video_utils.py

```diff
@@ -158,22 +162,52 @@ def decode_video_frames_torchvision(
     return closest_frames
 
 
-def encode_video_frames(imgs_dir: Path, video_path: Path, fps: int):
-    """More info on ffmpeg arguments tuning on `lerobot/common/datasets/_video_benchmark/README.md`"""
+def encode_video_frames(
+    imgs_dir: Path,
+    video_path: Path,
+    fps: int,
+    video_codec: str = "libsvtav1",
+    pixel_format: str = "yuv420p",
+    group_of_pictures_size: int | None = 2,
+    constant_rate_factor: int | None = 30,
+    fast_decode: int = 0,
```

Comment:
We have `fast_decode` here, but it's commented out elsewhere. Is it expected?

Reply:
Yes, I was going to add it to the benchmark but then decided it's okay not to use it for now, although it could still be useful later so I'd leave it commented out for now.